addons/account/models/chart_template.py

```diff
@@ -1031,13 +1039,12 @@ def mapping_getter(*args):
 
     def _deref_account_tags(self, template_code, tax_data):
         mapper = self._get_tag_mapper(template_code)
-        for tax in tax_data.values():
-            for fname in ('invoice_repartition_line_ids', 'refund_repartition_line_ids', 'repartition_line_ids'):
-                if tax.get(fname):
-                    for _command, _id, repartition in tax[fname]:
-                        tags = repartition.get('tag_ids')
-                        if isinstance(tags, str):
-                            repartition['tag_ids'] = [Command.set(mapper(*tags.split(TAX_TAG_DELIMITER)))]
+        for tax_values in tax_data.values():
+            for field_name in ('repartition_line_ids', 'invoice_repartition_line_ids', 'refund_repartition_line_ids'):
+                for element in tax_values.get(field_name, []):
+                    match element:
+                        case Command.CREATE, 0, {'tag_ids': str() as tags} as values:
```

Comment:
Probably it's because I'm not into the `UPDATE` flow.
Is it when we like "reload" the chart template?
I'll recreate the flow and see what's missing 👍🏻

Reply:
It's for if a third party module want to update some of the lines (i.e. new field) without creating a brand new line

Reply:
Not all records are available for update though.

With `tax.tax_repartition_line_ids.tag_ids` as we actually rename the old tax and replace the tags, their `xmlid` becomes obsolete, and we never convert Command.CREATE to Command.UPDATE and that I think is the only way to get an update command.

I have to try with like an `account.fiscal.position` that adds taxes I think. Or some other x2many field. 

Is it correct?